tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/bytetype/BByteValueTest.java

```diff
@@ -792,4 +792,34 @@ private void invokeBitwisePrecedenceTestFunctionForInt(int a, int b, int c, Stri
         BInteger bInteger = (BInteger) returns[0];
         Assert.assertEquals(bInteger.intValue(), expected, "Invalid byte value returned.");
     }
+
+    @Test(description = "Test byte value return as int in lambda 1")
+    public void testByteReturnAsIntInLambda1() {
+        BValue[] returns = BRunUtil.invoke(result, "testByteReturnAsIntInLambda1");
```

Comment:
With new lang tests we've tried writing the tests in Ballerina itself as much as possible.

For example https://github.com/ballerina-platform/ballerina-lang/blob/master/tests/jballerina-unit-test/src/test/resources/test-src/expressions/listconstructor/list_constructor_infer_type.bal#L55

We can rewrite these tests similarly right?

Reply:
looks like you have to repeat the assertEquality method for each new test ?

Reply:
And yes, we can write that in the new style. The thing is all the other methods are written in the old way. May be we have to add a new bal test file with these new tests.

Reply:
For the time-being yes, the `assertEquality` function has to be repeated in each test file. But once the compiler test suite is done we can directly switch to calling a similar (most-likely the same) function in that particular module.

Reply:
Yeah, there is some inconsistency when adding tests to an existing file. But even then, IMO it's kinda OK given that we usually add new tests at the end?

I'm OK with either approach.

Reply:
Updated the tests :)

Reply:
Great, thanks!